include/fmt/std.h

```diff
@@ -28,6 +31,16 @@
 #  endif
 #endif
 
+#if FMT_HAS_INCLUDE(<cxxabi.h>) || defined(__GLIBCXX__)
```

Comment:
Why do we need `__GLIBCXX__` check? Isn't `FMT_HAS_INCLUDE` enough?

Reply:
GCC 4 does not support FMT_HAS_INCLUDE, but it does support demangling.
It is possible to check for the presence of libstdc++ instead of gcc version, since all libstdc++ support demangling.

Reply:
Could you add a comment that the `__GLIBCXX__` is for gcc 4 here then?

Reply:
Done